Automattic/jetpack
# Admin Page: Update  gulp-sass to version 3.1.0

Fixes build on sandboxes

#### Changes proposed in this Pull Request:

* Updates `gulps-sass` to version 3.1.0 for which node-sass has a binary available for most platforms.

#### Testing instructions:

* Checkout this branch.
* Run `yarn distclean && yarn build`
* Confirm that the build ends successfully

* [ ] Builds on Sandbox
* [x] Builds on MacOs Sierra
* [ ] Builds on MacOs High Sierra
* [ ] Builds on Ubuntu
